### ISSUE TITLE
x/tools/cmd/godoc: proposed new template design

### ISSUE THREAD

**Issue by gopherbot** [golang/go#3496]
by **milak.mlk**:

<pre>This is not a bug but a enhencement proposed.
I think the godoc is not easy to browse. So here is my own template.
Thanks for you work.</pre>

Attachments:
1. <a href="https://storage.googleapis.com/go-attachment/3496/0/package.html">package.html</a> (7468 bytes)

**Comment by agnivade** [comment]
In light of the new redesign of godoc, I believe this is redundant now. We can open another issue if needed, once the redesign is complete.

Closing this.
